### PR TITLE
Updates to feature icons

### PR DESCRIPTION
**Pull request by ina-stoyanova**
New:
Screenshot 2021-08-25 at 02 08 15
 vs Old:
Screenshot 2021-08-25 at 02 08 29

### CODE REVIEW

**Comment by ina-stoyanova** [comment]
I don't like the fact the spacing is different for the different icons, so I'll look into this tomorrow.

File: pages/landing/patcher/_features.html [hunk 1]
```diff
@@ -3,8 +3,8 @@ <h2>Patcher Features</h2>
   {% for feature in page.features %}
     <div class="col-xs-6 col-md-4" style="margin-bottom:20px;">
       <div class="row" style="display: flex; flex-wrap: wrap;">
-        <div class="col-xs-12 col-sm-4 hidden-print" style="padding-right: 0; padding-left: 0;">
-          <img src="{{ feature.icon }}" alt="{{ feature.title }}" style="max-width: 105px;" />
+        <div class="col-xs-12 col-sm-3 hidden-print" style="padding-right: 5px;">
```
Comment (brikis98) [inline pages/landing/patcher/_features.html:6]:
```suggestion
        <div class="col-xs-12 col-sm-3 hidden-print" style="padding-right: 5px; text-align: right">
```

If you add this to each icon, the spacing issues will be fixed

File: pages/landing/patcher/_features.html [hunk 2]
```diff
@@ -3,8 +3,8 @@ <h2>Patcher Features</h2>
...
+          {{ feature.icon }}
         </div>
         <div class="col-xs-12 col-sm-8" style="line-height: 1.4;">
           <h4 class="h6 text-bold" style="margin: 5px 0 5px 0">{{ feature.title }}</h4>
```
Comment (brikis98) [inline pages/landing/patcher/_features.html:10]:
```suggestion
          <h4 class="h6 text-bold" style="margin: 0">{{ feature.title }}</h4>
```

If you get rid of the margin for each feature description, the headings for each feature will align with the top of the icons better too.

File: pages/landing/patcher/index.html [hunk 3]
```diff
@@ -101,27 +101,27 @@
 features:
   - title: Update dependency versions
     description: Like DependaBot, Patcher automatically updates you to the latest versions of dependencies
-    icon: /assets/img/landing-zone-features/features-7@3x.png
+    icon: '<i class="fa fa-refresh fa-4x" aria-hidden="true"></i>'
```
Comment (brikis98) [inline pages/landing/patcher/index.html:104]:
```suggestion
    icon: '<i class="fa fa-refresh fa-3x" aria-hidden="true"></i>'
```

NIT: total matter of preference, but I _think_ the icons look a tiny bit better one size smaller.

File: pages/landing/patcher/index.html [hunk 4]
```diff
@@ -101,27 +101,27 @@
 features:
   - title: Update dependency versions
     description: Like DependaBot, Patcher automatically updates you to the latest versions of dependencies
```
Comment (brikis98) [inline pages/landing/patcher/index.html:103]:
```suggestion
    description: Patcher automatically updates you to the latest versions of dependencies
```

File: pages/landing/patcher/index.html [hunk 5]
```diff
@@ -101,27 +101,27 @@
...
-    icon: /assets/img/landing-zone-features/features-7@3x.png
+    icon: '<i class="fa fa-refresh fa-4x" aria-hidden="true"></i>'
 
   - title: Handle breaking changes
     description: Unlike DependaBot, Patcher can also patch your code to handle backwards incompatible changes
```
Comment (brikis98) [inline pages/landing/patcher/index.html:107]:
```suggestion
    description: Patcher can also patch your code to handle backwards incompatible changes
```

File: pages/landing/patcher/_features.html [hunk 6]
```diff
@@ -3,8 +3,8 @@ <h2>Patcher Features</h2>
   {% for feature in page.features %}
     <div class="col-xs-6 col-md-4" style="margin-bottom:20px;">
       <div class="row" style="display: flex; flex-wrap: wrap;">
-        <div class="col-xs-12 col-sm-4 hidden-print" style="padding-right: 0; padding-left: 0;">
-          <img src="{{ feature.icon }}" alt="{{ feature.title }}" style="max-width: 105px;" />
+        <div class="col-xs-12 col-sm-3 hidden-print" style="padding-right: 5px;">
+          {{ feature.icon }}
         </div>
         <div class="col-xs-12 col-sm-8" style="line-height: 1.4;">
```
Comment (brikis98) [inline pages/landing/patcher/_features.html:9]:
```suggestion
        <div class="col-xs-12 col-sm-8" style="line-height: 1.4; padding-right: 0">
```

NIT: this allows the heading "Update dependency versions" to fit on one line, which looks a tiny bit cleaner.

**Review by brikis98 — COMMENTED** [review]
Nice improvement! I made a few suggestions on how to clean the spacing up a little bit 👍

File: pages/landing/patcher/index.html [hunk 7]
```diff
@@ -100,28 +100,28 @@
...
-    description: Like DependaBot, Patcher automatically updates you to the latest versions of dependencies
-    icon: /assets/img/landing-zone-features/features-7@3x.png
+    description: Patcher automatically updates you to the latest versions of dependencies
+    icon: '<i class="fa fa-refresh fa-3x" aria-hidden="true"></i>'
 
   - title: Handle breaking changes
-    description: Unlike DependaBot, Patcher can also patch your code to handle backwards incompatible changes
-    icon: /assets/img/landing-zone-features/features-1@3x.png
+    description: Patcher can also patch your code to handle backwards incompatible changes
+    icon: '<i class="fa fa-bolt fa-4x" aria-hidden="true"></i>'
```
Comment (brikis98) [inline pages/landing/patcher/index.html:108]: Note: you need to make the others 3x as well, not just the first!

Comment (ina-stoyanova) [reply pages/landing/patcher/index.html:108]: Oh, i did..... but forgot to push my change!

Comment (ina-stoyanova) [reply pages/landing/patcher/index.html:108]: thanks for noticing! pushing it now!

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by ina-stoyanova** [comment]
Oh, hold on - so you were able to still approve despite me merging your suggestions? That's awesome! Thanks for the help & review! :)

**Pull request merged**